holoviews/plotting/mpl/renderer.py

```diff
@@ -318,3 +321,4 @@ def load_nb(cls, inline=True):
         """
```

Comment:
An attribute is fine but I was thinking of a parameter. Maybe a property makes the most sense as it is read-only by default?